scripts/deploy_limited_whitelisted_recipients.py

```diff
@@ -0,0 +1,161 @@
+from brownie import chain, network
+
+from utils.config import (
+    get_env,
+    get_is_live,
+    get_deployer_account,
+    prompt_bool,
+    network_name
+)
+from utils import (
+    deployment,
+    lido,
+    deployed_easy_track,
+    log
+)
+
+from brownie import (
+    WhitelistedRecipientsRegistry,
+    AddWhitelistedRecipient,
+    RemoveWhitelistedRecipient,
+    TopUpWhitelistedRecipients
+)
+
+def main():
+    netname = "goerli" if network_name().split('-')[0] == "goerli" else "mainnet"
+
+    contracts = lido.contracts(network=netname)
+    et_contracts = deployed_easy_track.contracts(network=netname)
+    deployer = get_deployer_account(get_is_live(), network=netname)
+
+    easy_track = et_contracts.easy_track
+    evm_script_executor = et_contracts.evm_script_executor
+
+    # address allowed to create motions to add, remove or top up whitelisted recipients
+    whitelisted_recipients_multisig = get_env("WHITELISTED_RECIPIENTS_MULTISIG")
```

Comment:
Looks like not the best naming here. There will be no special WHITELISTED_RECIPIENTS_MULTISIG, there will be different committee multisigs that will own their separate registries. And, by design, it can be EOA or other contract

So, I'd rather stick with COMMITTEE_MULTISIG or owner or something not related to whitelist

Reply:
Thanks. replaced by COMMITTEE_MULTISIG